### PR TITLE
Update references to Model docs

### PR DESCRIPTION
**Pull request by hflynn**
See https://github.com/ome/ome-model/pull/19 for context.

Staged at https://www.openmicroscopy.org/site/support/contributing-staging/ and https://www.openmicroscopy.org/site/support/contributing-staging/ci-docs.html

### CODE REVIEW

**Comment by hflynn** [comment]
This still references the generation of pdfs which is incorrect for 5.3.x on BF and the new Model docs but as it's all one replacement, I can't edit without removing it for everything.

File: contributing/ci-docs.txt [hunk 1]
```diff
@@ -101,7 +101,7 @@ are then used:
   ``-W -D html_theme=sphinx_theme``,
 
 - the release number of the documentation is set by :envvar:`OMERO_RELEASE`,
-  :envvar:`BF_RELEASE` or :envvar:`FORMATS_RELEASE`,
+  :envvar:`BF_RELEASE` or :envvar:`MODEL_RELEASE`,
```
Comment (rleigh-codelibre) [inline contributing/ci-docs.txt]: If this is referring to the ome-model logic, it's set in the maven root POM [here](https://github.com/ome/ome-model/blob/a146b61c3dd5c624cebb24b499bcd9f980165f2b/pom.xml#L121) and then used to make the sphinx conf.py [here](https://github.com/ome/ome-model/blob/a146b61c3dd5c624cebb24b499bcd9f980165f2b/docs/sphinx/pom.xml#L37).  There's no MODEL_RELEASE in ome-model, or is this elsewhere?

File: contributing/ci-docs.txt [hunk 2]
```diff
@@ -286,14 +288,14 @@ sets is develop.
...
-	:jenkinsjob:`FORMATS-latest-docs`
+	:jenkinsjob:`MODEL-DEV-latest-docs`
 
 		This job is used to build the develop branch of the OME Model
 		documentation and publish the official documentation
 
 		#. |sphinxbuild|
 		#. |linkcheck|
-		#. |ssh-doc| :file:`formats-release.tmp`
+		#. |ssh-doc| :file:`model-release.tmp`
```
Comment (rleigh-codelibre) [inline contributing/ci-docs.txt]: `ome-model-release.tmp`

File: contributing/ci-docs.txt [hunk 3]
```diff
@@ -259,20 +259,22 @@ The branch for the 5.3.x series of the Bio-Formats documentation is develop.
...
+	:jenkinsjob:`MODEL-DEV-merge-docs`
 
 		This job is used to review the PRs opened against the develop branch
 		of the OME Model documentation
 
 		#. |merge|
 		#. |sphinxbuild|
 		#. |linkcheck|
-		#. |ssh-doc| :file:`formats-staging.tmp`
+		#. |ssh-doc| :file:`model-staging.tmp`
```
Comment (rleigh-codelibre) [inline contributing/ci-docs.txt]: `ome-model-staging.tmp`

File: contributing/ci-docs.txt [hunk 4]
```diff
@@ -259,20 +259,22 @@ The branch for the 5.3.x series of the Bio-Formats documentation is develop.
 OME Model and OME Contributing
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The branch for the OME model and the OME Contributing documentation
-sets is develop.
+The branch for the OME Model and the OME Contributing documentation
```
Comment (rleigh-codelibre) [inline contributing/ci-docs.txt]: The branch for the OME data model is `master`.  I think this needs to be in a separate sentence from the contributing documentation.  They are in separate places, on separate branches, and are completely unrelated.

Comment (hflynn) [reply contributing/ci-docs.txt]: Yeah, I'll clean that up thanks

**Comment by hflynn** [comment]
Should be fixed now @rleigh-codelibre although the merge build on Jenkins won't be green until the latest job is created too.

**Comment by rleigh-codelibre** [comment]
https://ci.openmicroscopy.org/view/Docs/job/MODEL-latest-docs/ created

**Comment by hflynn** [comment]
👍 I'll re-run the merge build then

**Comment by hflynn** [comment]
https://ci.openmicroscopy.org/view/Docs/job/CONTRIBUTING-merge-docs/949/ is indeed green now

File: contributing/ci-docs.txt [hunk 5]
```diff
@@ -259,20 +259,21 @@ The branch for the 5.3.x series of the Bio-Formats documentation is develop.
 OME Model and OME Contributing
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The branch for the OME model and the OME Contributing documentation
-sets is develop.
+The OME Contributing documentation is located in the ome-documentation
+repository and is built from the develop branch. The Model documentation is
+now located in the ome-model repository and is built from the master branch.
```
Comment (rleigh-codelibre) [inline contributing/ci-docs.txt]: Do we need the "now" here?  (to save editing it again in the future)

**Comment by rleigh-codelibre** [comment]
This all looks good to merge; one really minor comment.

**Comment by hflynn** [comment]
Good point, I'll remove that and then merge, thanks.

**Pull request merged**